### DISCUSSION TITLE
Claim Airdop, Trading and Voting have been marked as malicious

### DISCUSSION THREAD

**Discussion by LossOf2021** [phantom/docs#356] [Help & Support] [answered]
### Summary

Our website https://call.meme is a token launchpad where users can create tokens with same and fair conditions.
there are 2 major functions:
1. creating tokens and trading
2. proposing/promoting tokens and voting (winner will receive funds) 
3. trading activities will get rewarded with platform airdrop.

And when users interact with our trading/voting/airdrop smart contract via phantom wallet 
it shows *We believe this transaction is maliciousand unsafe to sign. Approving may leadto loss of funds.  We've contact with blowfish for our NFT issue a couple days ago and it is resolved by them. When we found them for issues above, they told us to contact Phantom team therefore I guess it is not their solution resolving this alert.

![warning](https://github.com/user-attachments/assets/2c4900ce-424f-4de9-a634-c45da7d04f87)
*

### Example

https://call.meme/tokensdetails/Hg4B6rZc2ozWCC3uyfJP8W3AWCQG3AvDh3NroZbv1DJW

### Steps to Reproduce

1. visit any function interact with smart contact e.g https://call.meme/tokensdetails/Hg4B6rZc2ozWCC3uyfJP8W3AWCQG3AvDh3NroZbv1DJW to buy token
2. connect phantom wallet

### Phantom Version

24.27.1 chrome extension

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment, marked as answer]
Hello @LossOf2021,

Apologies for the delay! Please reach out to review@blowfish.xyz if you have not already.
